meta/main.yml

```diff
@@ -5,7 +5,7 @@ galaxy_info:
   company: pro!vision
   issue_tracker_url: https://wcm-io.atlassian.net
   license: Apache
-  min_ansible_version: 2.4
+  min_ansible_version: 2.6
```

Comment:
Looking at the [module code ](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/amazon/aws_s3.py) it seems like it was renamed in 2.4, so do we really need that min. version bump?

Reply:
True. According to Ansible Docu: https://docs.ansible.com/ansible/latest/modules/aws_s3_module.html

> In 2.4, this module has been renamed from s3 into aws_s3.

will change the min version back to 2.4.

Reply:
done. corrected the ansible version also in README